contentful-data/rollbacks/prod-export.js

```diff
@@ -0,0 +1,30 @@
+function migrationFunction(migration, context) {
+    const test = migration.createContentType("test");
+    test
+        .displayField("test")
+        .name("test")
+        .description("test")
+
+    const testTest = test.createField("test");
```

Comment:
Is there a better name we can give this?

Reply:
These scripts are auto-generated by the contentful migration commands, the 'test' fields are just referring to content that I created while I was testing the migration scripts. They should be overwritten the next time the script is run